src/platform/macos.rs

```diff
@@ -36,6 +36,9 @@ pub trait WindowExtMacOS {
 
     /// Sets whether or not the window has shadow.
     fn set_has_shadow(&self, has_shadow: bool);
+
+    /// Returns the current window theme.
+    fn theme(&self) -> Theme;
```

Comment:
> Could we not do that as a macOS extension? We have theme setting on like every platform

If I do this, I need to implement theme method for getting current theme on unix?
I think this feature is not implemented on unix platform.

> And should likely return Option in that case, since we could fail identify it? Like on old macOS.

This should be handled by https://github.com/rust-windowing/winit/pull/2441/files#diff-4803bb2335a17d700335730b9f62828e001faa602b9b894dff572422c737b78aR1364-R1367.